### PR TITLE
[pangolin] Fix build failure on UNIX

### LINKED ISSUE #10099: [pangolin] build failure
**Host Environment**
- OS: Ubuntu 18.04
- Compiler: gcc 7.4.0

**To Reproduce**
`./vcpkg install pangolin:x64-linux`

**Failure logs** 
Look at an attachment
[install-x64-linux-dbg-out.log](https://github.com/microsoft/vcpkg/files/4211074/install-x64-linux-dbg-out.log)

### PR DESCRIPTION
**Pull request by JackBoosY**
Finding ffmpeg components in Pangolin's built-in `FindFFmpeg.cmake` is not complete, which causes the build failure in UNIX.

Fixes #10099

Thanks @Wkkkkk!

### CODE REVIEW

File: scripts/ci.baseline.txt [hunk 1]
```diff
@@ -1304,9 +1304,6 @@ osgearth:x64-windows-static=fail
 osg-qt:x64-windows-static=fail
 paho-mqtt:arm-uwp=fail
 paho-mqtt:x64-uwp=fail
-pangolin:x64-linux=fail
-pangolin:x64-osx=fail
-pangolin:x64-uwp=fail
 pangolin:x64-windows-static=fail
```
Comment (PhoebeHui) [inline scripts/ci.baseline.txt:1307]: Could you also add the this to supports and remove it in ci.baseline.txt?

**Review by PhoebeHui — CHANGES_REQUESTED** [review]

**Review by PhoebeHui — APPROVED** [review]

File: ports/pangolin/CONTROL [hunk 2]
```diff
@@ -1,5 +1,7 @@
 Source: pangolin
-Version: 0.5-7
+Version: 0.5
+Port-Version: 8
 Build-Depends: eigen3, glew, libpng, libjpeg-turbo, ffmpeg
 Homepage: https://github.com/stevenlovegrove/Pangolin
 Description: Lightweight GUI Library
+Supports: !(uwp|osx)&!(windows&static)
```
Comment (strega-nil) [inline ports/pangolin/CONTROL:7]: This is a very confusing supports paragraph. Could we rewrite this as:

```suggestion
Supports: !uwp & !osx & !(windows&static)
```

**Review by strega-nil — CHANGES_REQUESTED** [review]
That's the only thing I see.

**Pull request merged**